parser-typechecker/src/Unison/Codebase/Causal.hs

```diff
@@ -135,69 +135,49 @@ lca a b =
   go Set.empty Set.empty (Seq.singleton $ pure a) . Seq.singleton $ pure b
```

Comment:
Not part of the PR, but I'd say the use of `.` here is probably counterproductive.